test/runit/tests/heatmap_downloadData.R

```diff
@@ -0,0 +1,35 @@
+source("web-app/HeimScripts/heatmap/downloadData.R")
+
+.setUp <- function() {
+  origDirectory <<- getwd()
```

Comment:
(unless you wanted to write to the parent scope, which might be the case since the variable is used only in this script ... )